pacemaker.spec.in

```diff
@@ -642,6 +645,12 @@ exit 0
 %postun cluster-libs -p /sbin/ldconfig
 %endif
 
+%post schemas
+xmlcatalog --noout --add delegateURI \
+  http://clusterlabs.org/nsredir/pacemaker/ \
+  file://%{_datadir}/pacemaker/catalog.xml \
+  %{_sysconfdir}/xml/catalog
```

Comment:
Indeed, I forgot about removal on uninstall, thanks.

Regarding the path, situation really varies across Fedora packages,
will take a closer look, but don't think it should be a critical point here.

Regarding guidelines, for long term, I've opened an idea of adopting
the openSUSE style in Fedora:
https://lists.fedoraproject.org/archives/list/packaging@lists.fedoraproject.org/message/5WP5NAC23XX6CBCDRHZVDA4BAHBEKCXH/

Picky downstream will need to adjust, and the more generic approach
will still work there, regardless.

Let me also think about `make install` and `make uninstall` repercussions,
since that's also a valid way to have the system installed.

Reply:
Ok, I think it's sufficiently resolved.

OpenSUSE is expected to pass `--with-detachedxmlcatalogdir=/etc/xml/catalog.d`
and run their devised commands in `%post` and `%postun`.